src/app/lib/analyticsUtils/staticATITracking/addProcessClientDeviceAndSendStaticBeaconToWindow/index.ts

```diff
@@ -15,27 +15,25 @@ export default () => {
       // COOKIE SETTINGS
       const cookieName = 'atuserid';
       const expires = 397; // expires in 13 months
-      const cookiesForPage = `; ${document.cookie}`;
-      const atUserIdCookie = cookiesForPage.split(`; ${cookieName}=`);
-      let user: { val: null | string } = { val: null };
 
-      if (atUserIdCookie.length === 2) {
-        const cookieInfo = atUserIdCookie.pop()?.split(';').shift();
-
-        if (cookieInfo) {
-          const decodedCookie = decodeURIComponent(cookieInfo);
-          user = JSON.parse(decodedCookie);
+      const getAtUserIdFromCookie = () => {
+        try {
+          const [, atUserIdCookie] =
+            document.cookie.match(/(?:^|;\s*)atuserid=([^;]*)/) || [];
+          return { val: JSON.parse(decodeURIComponent(atUserIdCookie))?.val };
```

Comment:
I think this makes it a bit more readable:

```suggestion
          const [, atUserIdCookie] =
            document.cookie.match(/(?:^|;\s*)atuserid=([^;]*)/) || [];
          
          const { val } =  JSON.parse(decodeURIComponent(atUserIdCookie)) || {};
          
          return { val };
```